kolibri/core/assets/src/views/content-icon/index.vue

```diff
@@ -97,6 +113,23 @@
       colorClass() {
         return `color-${this.colorStyle}`;
       },
+      tooltipText() {
+        const kind = this.kind;
```

Comment:
Missing channel, lesson, exam, and user.

Reply:
thanks - updated

Reply:
Last one is USER